composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/screens/RegisterScreen.kt

```diff
@@ -180,7 +180,7 @@ fun ProfileCompletionStep(
             trailingIcon = {
                 ClearFieldIcon { onAction(RegisterScreenUiAction.OnFirstNameChangedAction("")) }
             },
-            errorMessage = if (state.isFirstNameValid) null else SharedRes.strings.first_name_validation_error.toStringResource()
+            errorMessage = if (state.isFirstNameValid != false) null else SharedRes.strings.first_name_validation_error.toStringResource()
```

Comment:
This is harder to read than above where u use isValid == false ... else null

Reply:
I would opt to change all of them in this pr to behave the samen with an == false ... else null